pkg/utils/test/predicates.go

```diff
@@ -592,6 +593,60 @@ func DataPlaneHasServiceSecret(t *testing.T, ctx context.Context, dpNN, usingSvc
 	}, clients.OperatorClient)
 }
 
+// PodDisruptionBudgetRequirement is a function type used to check if a PodDisruptionBudget meets a certain requirement.
+type PodDisruptionBudgetRequirement func(policyv1.PodDisruptionBudget) bool
+
+// AnyPodDisruptionBudget returns a function that accepts any PodDisruptionBudget.
+func AnyPodDisruptionBudget() PodDisruptionBudgetRequirement {
+	return func(policyv1.PodDisruptionBudget) bool {
+		return true
+	}
+}
+
+// DataPlaneHasPodDisruptionBudget is a helper function for tests that returns a function
+// that can be used to check if a DataPlane has a PodDisruptionBudget.
```

Comment:
Can we expand this comment to explain what `has` means here? Does that mean that a PDB is created in the cluster? That it has all the required metadata (ownership, labels etc)?